src/test/regress/sql/distributed_types_xact_add_enum_value.sql

```diff
@@ -0,0 +1,33 @@
+SET citus.next_shard_id TO 20040000;
+
```

Comment:
Now that we have differing test outputs, I think we should print PG versions in these tests now:

```
SHOW server_version \gset
SELECT substring(:'server_version', '\d+')::int > 11 AS version_above_eleven;
```

Reply:
Good idea, have it added